### ISSUE TITLE
Support for the new codeclimate config file

### ISSUE THREAD

**Issue by daften** [codeclimate/codeclimate#763]
According to https://docs.codeclimate.com/docs/advanced-configuration there's a new codeclimate.yml configuration file format.

When I try the one below in package https://github.com/digipolisgent/robo-digipolis-general, I get 

```
blommedi at macbook-pro in ~/Workspaces/infrastructuur/robo/robo-digipolis-general on develop [!]
$ codeclimate analyze
No enabled engines. Add some to your .codeclimate.yml file!
```

```
version: "2"
plugins:
  phpcodesniffer:
    enabled: true
    config:
      standard: "PSR1,PSR2"
  phpmd:
    enabled: true
    config:
      CleanCode/StaticAccess:
        enabled: false
      Naming/ShortVariable:
        enabled: false
      Naming/ShortMethodName:
        enabled: false
  duplication:
    enabled: true
    config:
      languages:
        - php:
  fixme:
    enabled: true
  markdownlint:
    enabled: true
    config:
      MD024:
        enabled: false
exclude_patterns:
  - "**/vendor/**/*"
  - "**/.idea/**/*"
  - "tests/"
```

**Comment by ale7714** [comment]
👋  @daften, thank you for trying out the new checks. The new maintainability checks are in beta but work is in progress to release them to the CLI. Unfortunately for now, the CLI doesn't support the new `codeclimate.yml` configuration. Sorry for the trouble and I'll update back as soon as I have new info.

**Comment by wfleming** [comment]
Hi @daften,

We released version 0.70.0 of the CLI today with support for the new configuration schema, as well as some significant new analysis checks. You can see some release notes at https://github.com/codeclimate/codeclimate/releases/tag/v0.70.0.

If you upgrade your CLI to get the latest version, the config file you posted above should work.

I'm going to go ahead and close this issue, but please re-open and @ me here if your config file still isn't working after upgrading.

**Comment by daften** [comment]
Thanks @wfleming for the quick fix and notification! :)
